tests/e2e/cucumber/steps/ui/public.ts

```diff
@@ -31,11 +32,10 @@ When(
   async function (this: World, stepUser: string, password: string): Promise<void> {
     const { page } = this.actorsEnvironment.getActor({ key: stepUser })
     const pageObject = new objects.applicationFiles.page.Public({ page })
-    const linkObject = new objects.applicationFiles.Link({ page })
     if (password === '%copied_password%') {
       password = await page.evaluate('navigator.clipboard.readText()')
     } else {
-      password = password === '%public%' ? linkObject.securePassword : password
+      password = '%public%' === password ? securePassword : password
```

Comment:
this is more preferable
```suggestion
      password = password === '%public%' ? securePassword : password
```